e2e/utils/tenderly.ts

```diff
@@ -1,5 +1,6 @@
 const TENDERLY_ACCESS_KEY = process.env['TENDERLY_ACCESS_KEY'];
-if (!TENDERLY_ACCESS_KEY) throw new Error('No TENDERLY_ACCESS_KEY provided in .env');
+if (!TENDERLY_ACCESS_KEY)
+  throw new Error('No TENDERLY_ACCESS_KEY provided in .env');
```

Comment:
nit: lets wrap this in brackets to improve readability

Reply:
Done